static/js/components/errorMessages.js

```diff
@@ -6,7 +6,7 @@ import ErrorMessage from "./ErrorMessage"
 
 export const UnableToLoadData = () => {
   // $FlowFixMe
```

Comment:
I think the `$FlowFixMe` here can be removed (I tried out removing it locally on my machine and flow passes ok with the change you made)